.gitignore

```diff
@@ -3,7 +3,8 @@ build
 fix_typos
 code_linter
 src/version/version.h
-
+run.sh
```

Comment:
While this is ok in this GSoC repository, this change would be unacceptable in the pgRouting repository.
You need to learn to live with the files that are "helper" files for the development you are doing, past present of future
For example, in my case I have zillions of files I have to see when I do a `git status`, and I live with them. Either I used them in the past, or I am using them, or I am preparing something for the future.
But none of them are to be included in the repository.


```
git status
On branch foo
Your branch is up to date with 'origin/foo'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	after.txt
	astar_benchmark.pg
	before.txt
	change.sh
	commitByDirectory.sh
	commitBySubDirectory.sh
	commits.sh
	contributions.txt
	contributors.txt
	dmain.txt
	errs.txt
	exe.sh
	get_contributors.sh
	info.txt
	install-all.sh
	install-old-version.sh
	install-one.sh
	instructions.sh
	nmain.txt
	node_change.txt
	orderedinfo.txt
	page_history_gh_page.js
	patch.cpp
	pge_history_main.js
	run.sh
	taptest-4.sh
	taptest.sh
	update3.4.sh
	v4.sh
```